### DISCUSSION TITLE
🔦 Feature: Create Nocodb template under dooqod.com

### DISCUSSION THREAD

**Discussion by artakvg** [nocodb/nocodb#7661] [Announcements]
### Please confirm if feature request does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Describe the usecase for the feature

It'll be much much easier to have some basic onboarding materials along with the real env already installed, so first commers can easily run it and try it.

### Suggested Solution

- Create necodb account under dooqod.com
- launch Vscode or just a terminal (they support almost any Linux machne) - this will be NocoDB Starter project
- add Youtube videos or other content you already think is valuable under NocoDB Starter project
- save as template, and publish

Usually  templates are accepted quickly, and sometimes dooqod.com team will handle setup for you.

**Comment by pkmkdennis** [comment]
plz, fix the function of sum()...thanks
